app/src/main/res/layout/content_manage_group.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks like a one-to-one copy of `content_main.xml`?

I would say just either throw `content_main.xml` directly in `activity_manage_group.xml` or, maybe cleaner, only put the RecyclerView in there